translationRecorder/app/src/main/java/org/wycliffeassociates/translationrecorder/Recording/RecordingActivity.java

```diff
@@ -426,6 +431,11 @@ private void addTakeToDb() {
                 0,
                 mUser.getId()
         );
+
+        if(chunkPlugin != null) {
+            ProjectProgress pp = new ProjectProgress(mProject, db, chunkPlugin.getChapters());
+            pp.updateProjectProgress();
+        }
         db.close();
```

Comment:
probably not good to close this if we're going to have one instance for the whole activity